extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -870,10 +860,8 @@ export class AmpA4A extends AMP.BaseElement {
    * @return {boolean}
    */
   isInNoSigningExp() {
-    return (
-      getExperimentBranch(this.win, NO_SIGNING_EXP.id) ===
-      NO_SIGNING_EXP.experiment
-    );
+    // eslint-disable-next-line no-undef
+    return NO_SIGNING_RTV;
```

Comment:
Do we need to cast it to boolean? (if undefined)

Reply:
Done.